extensions/data-workspace/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Turn on strict compiling for new projects

Reply:
it has been specified in the shared.tsconfig.json which is being referenced here